rollbar-cli/rollbar-cli.cabal

```
@@ -1,13 +1,13 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.36.0.
+-- This file has been generated from package.yaml by hpack version 0.38.1.
 --
 -- see: https://github.com/sol/hpack
 --
--- hash: 99c3aa74e5ef6c059b7ebeb837bca8f09c047df2a00b6856c99a919e4ad041ed
+-- hash: 06008e94a54478aab8ebf7a602178874866a988ec726e4bf4d2d6da0de509848
 
 name:           rollbar-cli
-version:        1.1.0
+version:        1.2.0
```

In Haskell we use [PVP](https://pvp.haskell.org/) for versioning, which is comprised by a four digit version _A.B.C(.D)_. When there are only three digits, we have the _A.B.C_ part. However, as opposed to semantic versioning, the _A.B_ part is what constitutes the major version:
> A.B is known as the major version number, and C the minor version number.

So this would be a major version bump. To do a minor version, we need to bump the _C_ part.

```suggestion
version:        1.1.1
```

---

Got it! Thanks for the heads up! @DavidMazarro I thought we were using semantinc versioning.